concepts/README.md

```diff
@@ -16,22 +16,30 @@ This directory contains various Dapr concepts. The goal of these documents is to
   
   Dapr uses a modular design, in which functionalities are grouped and delivered by a number of *components*, such as  [pub/sub](./publish-subscribe-messaging/README.md) and [secrets](./components/secrets.md). Many of the components are pluggable so that you can swap out the default implementation with your custom implementations. 
 
+* [**Configuration**](./configuration/README.md)
+
+  Dapr configuration defines a policy that affects how a Dapr sidecar hebavies, such as [distributed tracing](distributed-tracing/README.md) and [custom pipeline](middleware/middleware.md).
+
 * [**Distributed Tracing**](./distributed-tracing/README.md)
 
   Distributed tracing collects and aggregates trace events by transactions. It allows you to trace the entire call chain across multiple services. Dapr integrates with [OpenTelemetry](https://opentelemetry.io/) for distributed tracing and metrics collection. 
 
-* [**Publish/Subscribe Messaging**](./publish-subscribe-messaging/README.md)
+* [**Middleware**](./middleware/middleware.md)
   
-  Pub/Sub is a loosely coupled messaging pattern where senders (or publishers) publishes messages to a topic, to which subscribers subscribe. Dapr natively supports the pub/sub pattern.
+  Dapr allows custom middleware to be plugged into the request processing pipeline. Middleware can perform additional actions on a request, such as authentication, encryption and message transformation before the request is routed to the user code, or before the request is returned to the client.
```

Comment:
Could you also clarify the sceanrios in which middleware is involved and not involved. eg.  As per my understanding, middleware is not involved on Triggering for an input binding and when event is delivered to user app for a subscribed topic.